Fixes and logging

prasadrane86 opened the pull request.

prasadrane86 committed:
diff file was not suppose to be added

prasadrane86 commented:
@akshaymishra There is a compile issue in main.c at line #81. dev is not defined.

akshaymishra commented:
Build bug. My error to commit without check.

I have resolved issue but also this driver is only for eeprom. Current
board has flash, wil enable driver for it and share later


On Tue, 28 Jul, 2020, 19:55 Prasad Rane, <notifications@github.com> wrote:

> @akshaymishra <https://github.com/akshaymishra> There is a compile issue
> in main.c at line #81. dev is not defined.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/akshaymishra/decode-proximity-hw/pull/2#issuecomment-665071516>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABYVF4IN6I2XWJES4WRTTDR53NTZANCNFSM4PKERXVQ>
> .
>

prasadrane86 committed:
Added supported for stdio on RTT

prasadrane86 committed:
Command added to enable debug logs

prasadrane86 committed:
fix: Random number generator was hangging

Set the start bit after every random number generation.
Debug logs added in random number genration function.

prasadrane86 committed:
Command for printing EPHIDs added

prasadrane86 committed:
Created a seperate module for utility functions.

print_hex function moved to this new utility module.

prasadrane86 committed:
Print functions shifted to utils module

prasadrane86 committed:
Fix: Internal device ID was set as BLE address

ble_hs_id_set_rnd() has a check to discard address if it is not random
(upper 2 bits should be either 00 or 11 for random addresses).
This resulted in the supplied random number not being set as the device's
BLE address and the chip's internal static device ID was used to form
the BLE address, reveling the devices identity.

prasadrane86 committed:
Renamed dp3t_print_ephids() to print_dp3t_ephids()

prasadrane86 committed:
Manufacturer name changed to DSPWorks

prasadrane86 committed:
Some Device info service characteristic need to be handled

Added a TODO comment to make a note of it.
